scripts/generateTemplates.js

```diff
@@ -235,16 +239,30 @@ function addGalleryEntry(settingsData, settingsName, settingsDescription, galler
 
 /** Extract category resource info to build gallery files */
 function getCategoryResourcesInfo(object, categoryResourcesMap, templatePath) {
-    const templateSplit = templatePath.split("\\");
-    const key = templateSplit[templateSplit.length - 1];
-    if (key && object && object[LanguagesMap[DefaultLang]]) {
-        const entry = object[LanguagesMap[DefaultLang]];
-        categoryResourcesMap[key] = {
-            "en": {
-                name: entry.name,
-                description: entry.description,
-                order: entry.order
-            }
+    if (object) {
+        if (object[LanguagesMap[DefaultLang]]) {
```

Comment:
nit: `LanguagesMap[DefaultLang]` gets evaluated a whole bunch of times , should it be extracted to a local?

Reply:
since this code will go away soon, I'll keep it as is